src/components/AppLayout/index.tsx

```diff
@@ -1110,7 +1110,7 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 	}
 
 	return (
-		<Layout className={className}>
+		<Layout className={`${className} overflow-x-hidden overflow-y-hidden`}>
```

Comment:
overflow none?

Reply:
woh cards swipe k time x and y direction se space leraha tha so overflow band kr diya